### ISSUE TITLE
building boost-vcpkg-helpers:x64-windows failed with: BUILD_FAILED

### ISSUE THREAD

**Issue by ctigrisht** [microsoft/vcpkg#29395]
I was trying to install `Wt`, this is a dependency for it. I am getting this error, is there a fix/workaround? Thanks

```
Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:273 (z_vcpkg_download_distfile_show_proxy_and_fail)
  ports/boost-vcpkg-helpers/portfile.cmake:4 (vcpkg_download_distfile)
  scripts/ports.cmake:147 (include)


error: building boost-vcpkg-helpers:x64-windows failed with: BUILD_FAILED
error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+boost-vcpkg-helpers
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[boost-vcpkg-helpers]+Build+error
Include '[boost-vcpkg-helpers] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2023-01-24-8a88d63f241d391772fbde69af9cab96c3c64c75
    vcpkg-scripts version: unknown
```

Logs show:
```
-- Downloading https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt -> boost_LICENSE_1_0.txt...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
Downloading https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt
warning: Download failed -- retrying after 1000ms
warning: Download failed -- retrying after 2000ms
warning: Download failed -- retrying after 4000ms
error: Failed to download from mirror set
error: https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt: WinHttpSendRequest failed with exit code 12002
error: https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt: WinHttpSendRequest failed with exit code 12002
error: https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt: WinHttpSendRequest failed with exit code 12002
error: https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt: WinHttpSendRequest failed with exit code 12002
[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(956): 
```

**Comment by FrankXie05** [comment]
Please make sure your network is good or the proxy is available, make sure you can access https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt normally
![image](https://user-images.githubusercontent.com/65999885/216501927-fb61d0d9-287b-4273-87f7-362371eef44c.png)

**Comment by ctigrisht** [comment]
> Please make sure your network is good or the proxy is available, make sure you can access https://raw.githubusercontent.com/boostorg/boost/boost-1.81.0/LICENSE_1_0.txt normally

For some reason it's blocked, I had to install a VPN to download the package

**Comment by FrankXie05** [comment]
@ctigrisht Make sure vcpkg passes through your proxy.

**Comment by ctigrisht** [comment]
> 

for some reason it's blocked by my ISP, so I had to use a VPN, all good now

**Comment by FrankXie05** [comment]
For ISPs in different regions, this is indeed a headache.  :(
